File: playground_series/s3e3/jl/baseline.jl
using CSV
using DataFrames
using Statistics

trn = CSV.read("./playground_series/s3e3/data/train.csv", DataFrame)
tst = CSV.read("./playground_series/s3e3/data/test.csv", DataFrame)

pred = mean(trn.:Attrition)

sub = DataFrame(
    id = tst.:id,
    Attrition = pred
)

CSV.write("./playground_series/s3e3/submissions/baseline.csv", sub)

File: playground_series/s5e5/baseline.py
import polars as pl
import seaborn as sns
import statistics

trn = pl.read_csv("playground_series/s5e5/data/train.csv")
tst = pl.read_csv("playground_series/s5e5/data/test.csv")

y = trn["Calories"]
pred = y.mean()

sub = pl.DataFrame({"id": tst["id"], "Calories": pred})

# write out
sub.write_csv("playground_series/s5e5/submissions/baseline.csv")
